deploy/operator/clickhouse-operator-install-template-rbac.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: clickhouse-operator
+  name: clickhouse-operator-$OPERATOR_NAMESPACE
```

Comment:
If namespace is not assigned, may it happen to get a malformed name 'clickhouse-operator-'?

Reply:
please look to clickhouse-operator-install-template-rbac.yaml Line 11
https://github.com/Altinity/clickhouse-operator/pull/474/files/6f2226df4d17c8e875c5290010e8efe4440bfa98#diff-026f50bcbf865698c1f9af86eaccc742R11

namespace must be defined
if OPERATOR_NAMESPACE is not defined it setup to `kube-system`

Reply:
![image](https://user-images.githubusercontent.com/105560/88376238-6b876a00-cdb6-11ea-94f6-3f95cda9637e.png)